<script>
try{
    let result=someUndefinedFunction();//this throw an error

}
catch(error)
{
  document.write("an error occured",error.message);//handle error
}
finally{
    document.write("this code <br>")
}




</script>